src/cart/swim/swim.c

```diff
@@ -924,8 +924,7 @@ swim_progress(struct swim_context *ctx, int64_t timeout_us)
 					ctx->sc_next_event = ctx->sc_deadline;
 				ctx_state = SCS_PINGED;
 			} else {
-				if (ctx->sc_next_tick_time < ctx->sc_next_event)
-					ctx->sc_next_event = ctx->sc_next_tick_time;
+				ctx->sc_next_event = MIN(ctx->sc_next_event, ctx->sc_next_tick_time);
```

Comment:
@liw I don't really understand the usage of `sc_next_tick_time`, can you please explain it for me?